cirq/experiments/t1_decay_experiment.py

```diff
@@ -75,7 +75,7 @@ def t1_decay(sampler: work.Sampler,
 
     # Cross tabulate into a delay_ns, false_count, true_count table.
     tab = pd.crosstab(results.delay_ns, results.output)
-    del tab.columns.name
+    tab = tab.rename_axis(None, axis="columns")
```

Comment:
Suggestion:

```python
tab.rename_axis(None, axis="columns", inplace=True)
```

On my workstation in place rename is 6x faster. 

```
In [26]: %timeit df2.rename_axis(None, axis="columns")
83.7 µs ± 4.83 µs per loop (mean ± std. dev. of 7 runs, 10000 loops each)

In [27]: %timeit df2.rename_axis(None, axis="columns", inplace=True)
13.9 µs ± 31.6 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)
```

Reply:
Good idea.  Done.

Reply:
You should not be storing the return value back in tab.

Reply:
Oops.  I also shouldn't be making quick fixes on Friday evening!  
Fixed.